new/LIT6045SerataSegdat.xml

```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6045SerataSegdat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሥርዓተ፡ ስግደት፡ በተፍጻሜተ፡ በዓለ፡ ፶፡ ቅድስት፡</title>
+                  <title xml:lang="eng" xml:id="t2">Order of services for the Rite of Adoration፡</title>
+                  <title xml:lang="eng" xml:id="t3">Services for the close of Lent</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The service is intended for the end of the Pentecost (or the beginning of the
+                  fast of the Apostles). The text is usually preserved as one of the homilies of <persName ref="PRS8121RetuaH">Rǝtuʿa Hāymānot</persName></p>
```

Comment:
I think you could add this information as a relation saws:isAttributedToAuthor as well (in <body> inside the <listRelation>)

Reply:
Well, if it is really attributed to him, of course